src/action/action_built.cpp

```diff
@@ -325,8 +325,8 @@ static std::size_t FindCFramePercent(const std::vector<CConstructionFrame> &cfra
 {
 	const auto it =
 		ranges::find_if(cframes, [&](const auto &frame) { return percent < frame.Percent; });
```

Comment:
I did not notice before that there is a (backported) ranges-header in here :-)
Nice